FMDB.podspec

```diff
@@ -26,24 +31,28 @@ Pod::Spec.new do |s|
   s.subspec 'standalone' do |ss|
     ss.xcconfig = { 'OTHER_CFLAGS' => '$(inherited) -DFMDB_SQLITE_STANDALONE' }
     ss.dependency 'sqlite3'
-    ss.source_files = 'src/fmdb/FM*.{h,m}'
-    ss.exclude_files = 'src/fmdb.m'
-  end
-
-  # build with FTS support and custom FTS tokenizer source files
+    ss.dependency 'FMDB/common'
+	end
+	
+  # build the latest stable version of sqlite3 
+	# with FTS support and custom FTS tokenizer source files
   s.subspec 'standalone-fts' do |ss|
```

Comment:
I *think* that you could add a sub-subspec to `s.subspec 'standalone' do |ss|` for FTS. Something like this:

```
s.subspec 'standalone' do |ss|
    ss.xcconfig = { 'OTHER_CFLAGS' => '$(inherited) -DFMDB_SQLITE_STANDALONE' }
    ss.dependency 'sqlite3'
    ss.dependency 'FMDB/common'
    ss.subspec 'FTS' do 'sss'
        sss.dependency 'sqlite3/fts'
        sss.source_files = 'src/extra/fts3/*.{h,m}'
    end
end
```
You'll have to verify though...